helpers/inbound/README.md

```diff
@@ -2,116 +2,48 @@
 
 ## Table of Contents
 
-* [Quick Start for Local Testing with Sample Data](#quick_start_local_sample)
-* [Quick Start for Local Testing with Real Data](#quick_start_local_real)
-* [Deploy to Heroku](#heroku)
-* [Code Walkthrough](#code_walkthrough)
+* [Example Usage](#example_usage)
 * [Testing the Source Code](#testing)
 * [Contributing](#contributing)
 
-<a name="quick_start_local_sample"></a>
-# Quick Start for Local Testing with Sample Data
-
-```bash
-git clone https://github.com/sendgrid/sendgrid-go.git
-cd sendgrid-go
-```
-
-Run the Inbound Parse listener in your terminal:
-
-```bash
-cd helpers/inbound/
-go run inbound.go
-```
-
-In another terminal, run the test data sender:
-
-```bash
-cd [path to sendgrid-go/helpers/inbound]
-go run inbound.go ./sample_data/default_data.txt http://127.0.0.1:8000/inbound
-```
-
-More sample data can be found [here](https://github.com/sendgrid/sendgrid-go/tree/master/helpers/inbound/sample_data).
-
-View the results in the first terminal.
-
-<a name="quick_start_local_real"></a>
-# Quick Start for Local Testing with Real Data
-
-[Setup your MX records.](https://sendgrid.com/docs/Classroom/Basics/Inbound_Parse_Webhook/setting_up_the_inbound_parse_webhook.html#-Setup) Depending on your domain name host, you may need to wait up to 48 hours for the settings to propagate.
-
-Run the Inbound Parse listener in your terminal:
-
-```bash
-git clone https://github.com/sendgrid/sendgrid-go.git
-go run inbound.go
-```
-
-In another terminal, use [ngrok](https://ngrok.com/) to allow external access to your machine:
-```bash
-ngrok http 8000
+<a name="example_usage"></a>
+# Example Usage
+
+```go
+package main
+
+import (
+    "fmt"
+    "log"
+    "net/http"
+
+    "github.com/sendgrid/sendgrid-go/helpers/inbound"
+)
+
+func inboundHandler(response http.ResponseWriter, request *http.Request) {
```

Comment:
The example should do something with the response, simply for completeness. Could just use the old code here:
```
// Twilio SendGrid needs a 200 OK response to stop POSTing
response.WriteHeader(http.StatusOK)
```